ep/build-velox/src/build_velox.sh

```diff
@@ -39,6 +40,10 @@ for arg in "$@"; do
     ENABLE_BENCHMARK=("${arg#*=}")
     shift # Remove argument name from processing
     ;;
+  --run_setup_script=*)
```

Comment:
@zhejiangxiaomai No directly related to this PR. As ```arg``` number grows, should we consider add a [help/usage method](https://opensource.com/article/19/12/help-bash-program)?

Reply:
Thanks, I can add it in next PR.